File: Count Occurrence of a Digit/count_occurrence_of_a_digit.c
#include <stdio.h>
#include <stdlib.h>
int main() {
    int n, d;
    printf("Enter a number: ");
    scanf("%d", &n);
    printf("Enter the digit to search for: ");
    scanf("%d", &d);
    int temp = abs(n);
    int count = 0;
    while (temp > 0) {
        if (temp % 10 == d) {
            count++;
        }
        temp = temp / 10;
    }
    printf("Occurrence of digit %d is: %d\n", d, count);
    return 0;
}

File: Count Occurrence of a Digit/count_occurrence_of_a_digit.py
n = int(input("Enter a number: "))
d = int(input("Enter the digit to search for: "))
temp = abs(n)
count = 0
while temp > 0:
    if temp % 10 == d:
        count += 1
    temp //= 10
print("Occurrence of digit", d, "is:", count)
